dhcpv6/dhcpv6message.go

```diff
@@ -65,14 +65,42 @@ func GetTime() uint32 {
 	return uint32((now.Nanoseconds() / 1000000000) % 0xffffffff)
 }
 
+// NewSolicit creates a new SOLICIT message without populating the CID field.
+func NewSolicit(modifiers ...Modifier) (DHCPv6, error) {
```

Comment:
How about passing a Client DUID as parameter so that it returns a valid SOLICIT?

Reply:
I asked him to intentionally skip the CID - this function will be used for packets where we want custom CIDs so they can be added either via modifier, or later by the user. However we still should specify an empty CID (e.g. DUID_LL with 00:00:00:00:00:00 link-local addr) so that the packet is still valid, even if with an invalid CID. It is easy to detect the null address in a pcap dump if the user forgets to set it.

In addition, we could add `NewSolicitWithCID` but I am generally against adding functions unless necessary